eucalyptus.spec

```diff
@@ -426,7 +431,11 @@ export CFLAGS="%{optflags}"
 
 # Eucalyptus does not assign the usual meaning to prefix and other standard
 # configure variables, so we can't realistically use %%configure.
+%if %{?el6}
 ./configure --with-axis2=%{_datadir}/axis2-* --with-axis2c=%{axis2c_home} --with-wsdl2c-sh=%{S:2} --enable-debug --prefix=/ --with-apache2-module-dir=%{_libdir}/httpd/modules --with-db-home=/usr/pgsql-9.2 --with-extra-version=%{release}
+%else
+./configure --with-axis2=%{_datadir}/axis2-* --with-axis2c=%{axis2c_home} --with-wsdl2c-sh=%{S:2} --enable-debug --prefix=/ --with-apache2-module-dir=%{_libdir}/httpd/modules --with-db-home=%{_prefix} --with-extra-version=%{release}
```

Comment:
@gholms postgresql-server admin commands are in /usr/bin so I've been using this in the postgresql-binaries.properties file when I have been working around this path issue:

euca.db.home=/usr

Maybe we should use --with-db-home=/bin here (or if you're trying to avoid hardcoding it, set it somewhere above and use variable substitution.)